webdriver/tests/bidi/network/remove_intercept/remove_intercept.py

```diff
@@ -41,13 +39,18 @@ async def test_remove_intercept(
 
     on_network_event = wait_for_event(f"network.{phase}")
 
+    await subscribe_events(events=["browsingContext.load"], contexts=[top_context["context"]])
+
+    browsing_context_load_events = []
+
+    async def on_browsing_context_load_event(method, data):
+        browsing_context_load_events.append(data)
+
+    remove_listener = bidi_session.add_event_listener("browsingContext.load", on_browsing_context_load_event)
+
     # Request to top_context should be blocked.
     # TODO(https://github.com/w3c/webdriver-bidi/issues/188): Use a timeout argument when available.
-    with pytest.raises(asyncio.TimeoutError):
-        await asyncio.wait_for(
-            asyncio.shield(bidi_session.browsing_context.navigate(context=top_context["context"], url=text_url, wait="complete")),
-            timeout=2.0,
-        )
+    await bidi_session.browsing_context.navigate(context=top_context["context"], url=text_url, wait="complete")
```

Comment:
The problematic change is actually that one. The triggered navigation will never complete due to the added but not handled intercept. As long as not every browser supports network interception we should leave the former code. I'm going to add a comment and will provide a patch via https://bugzilla.mozilla.org/show_bug.cgi?id=1866267.